RELEASE.md

```diff
@@ -56,6 +56,9 @@ On master branch:
 
 After the release
 =================
-1. Create Windows installer
+1. Create Windows installer.
+Prerequisites: install Wix Toolset v.3.11 availabe here https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm
```

Comment:
Instead of pasting raw URLs, please use MD links, i.e. `[available here](https://github.com/wixtoolset/wix3/releases/tag/wix3112rtm)`.